src/main/fc/fc_tasks.c

```diff
@@ -211,7 +212,7 @@ static void taskTelemetry(timeUs_t currentTimeUs)
 // Everything that listens to VTX devices
 void taskVtxControl(uint32_t currentTime)
 {
-    if (ARMING_FLAG(ARMED))
+    if (ARMING_FLAG(ARMED) || cliMode)
         return;
 
 #ifdef VTX_COMMON
```

Comment:
Change `vtxCommonProcess(currentTime);` to `vtxCommonProcess(currentTime,vtxSettingsConfig());`

Reply:
If this vtxCommonProcess is moved to the io layer, is it necessary to pass the PG if it is already defined as part of the io code?  Seems like it is already native there.

Reply:
Agreed. Above comment was written before I noticed `vtxCommonProcess` should be moved to IO layer.